rust/lance-encoding/src/statistics.rs

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+use std::collections::HashMap;
+
+use arrow_array::ArrayRef;
+
+// A temporary and overly simplified sketch for arrow array statics,
+// The data structures and trait signiture here are totally temporary and of no use.
+// It is here only for filling in the `CoreArrayEncodingStrategy` field. 
+#[derive(Debug, Default)]
+pub struct Stats {
+    _stats_hashmap: HashMap<String, String>,
+}
+
+impl Stats {
+    fn new() -> Self {
+        Self {
+            _stats_hashmap: HashMap::new(),
+        }
+    }
+
+    pub fn inner_stats(&self) -> Stats {
```

Comment:
What's the difference between "inner stats", "key stats", "value stats", etc?  I'm not entirely sure I follow.

Reply:
oh, they are just placeholders for now, the names here are only because `fixed_size_list` can have `inner_type`, `dictionary_array` have `keys` and `values`, nothing meaningful here, sorry for the confusion.